### ISSUE TITLE
'GenerateTypeCodeFixProvider' encountered and error and has been disabled

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#51226]
StackTrace:

```
System.NullReferenceException : Object reference not set to an instance of an object.
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.PropertyGenerator.UseExpressionBodyIfDesired(CodeGenerationOptions options,AccessorDeclarationSyntax declaration,ParseOptions parseOptions)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.PropertyGenerator.GenerateAccessorDeclaration(IPropertySymbol property,IMethodSymbol accessor,SyntaxKind kind,Boolean hasBody,CodeGenerationOptions options,ParseOptions parseOptions)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.PropertyGenerator.GenerateAccessorDeclaration(IPropertySymbol property,IMethodSymbol accessor,SyntaxKind kind,CodeGenerationDestination destination,CodeGenerationOptions options,ParseOptions parseOptions)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.PropertyGenerator.GenerateAccessorList(IPropertySymbol property,CodeGenerationDestination destination,CodeGenerationOptions options,ParseOptions parseOptions)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.PropertyGenerator.GeneratePropertyDeclaration(IPropertySymbol property,CodeGenerationDestination destination,CodeGenerationOptions options,ParseOptions parseOptions)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.PropertyGenerator.GeneratePropertyOrIndexer(IPropertySymbol property,CodeGenerationDestination destination,CodeGenerationOptions options,ParseOptions parseOptions)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.PropertyGenerator.AddPropertyTo(TypeDeclarationSyntax destination,IPropertySymbol property,CodeGenerationOptions options,IList`1 availableIndices)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.CSharpCodeGenerationService.AddProperty[TDeclarationNode](TDeclarationNode destination,IPropertySymbol property,CodeGenerationOptions options,IList`1 availableIndices)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.UpdateDestination[TDeclarationNode](IList`1 availableIndices,CodeGenerationOptions options,TDeclarationNode currentDestination,ISymbol member,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.AddMembersToAppropiateLocationInDestination[TDeclarationSyntax](TDeclarationSyntax destination,IEnumerable`1 members,IList`1 availableIndices,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.AddMembers[TDeclarationNode](TDeclarationNode destination,IEnumerable`1 members,IList`1 availableIndices,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.AddMembers[TDeclarationNode](TDeclarationNode destination,IEnumerable`1 members,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.NamedTypeGenerator.GenerateNamedTypeDeclaration(ICodeGenerationService service,INamedTypeSymbol namedType,CodeGenerationDestination destination,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.NamedTypeGenerator.AddNamedTypeTo(ICodeGenerationService service,NamespaceDeclarationSyntax destination,INamedTypeSymbol namedType,CodeGenerationOptions options,IList`1 availableIndices,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.CSharpCodeGenerationService.AddNamedType[TDeclarationNode](TDeclarationNode destination,INamedTypeSymbol namedType,CodeGenerationOptions options,IList`1 availableIndices,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.UpdateDestination[TDeclarationNode](IList`1 availableIndices,CodeGenerationOptions options,TDeclarationNode currentDestination,ISymbol member,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.AddMembersToAppropiateLocationInDestination[TDeclarationSyntax](TDeclarationSyntax destination,IEnumerable`1 members,IList`1 availableIndices,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.AddMembers[TDeclarationNode](TDeclarationNode destination,IEnumerable`1 members,IList`1 availableIndices,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.AddMembers[TDeclarationNode](TDeclarationNode destination,IEnumerable`1 members,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.NamespaceGenerator.GenerateNamespaceDeclaration(ICodeGenerationService service,INamespaceSymbol namespace,CodeGenerationOptions options,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.NamespaceGenerator.AddNamespaceTo(ICodeGenerationService service,CompilationUnitSyntax destination,INamespaceSymbol namespace,CodeGenerationOptions options,IList`1 availableIndices,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CSharp.CodeGeneration.CSharpCodeGenerationService.AddNamespace[TDeclarationNode](TDeclarationNode destination,INamespaceSymbol namespace,CodeGenerationOptions options,IList`1 availableIndices,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.<>c__DisplayClass53_0.<AddNamespaceAsync>b__0(SyntaxNode t,CodeGenerationOptions opts,IList`1 ai,CancellationToken ct)
   at async Microsoft.CodeAnalysis.CodeGeneration.AbstractCodeGenerationService.GetEditAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.GenerateType.AbstractGenerateTypeService`6.Editor.GetGenerateInNewFileOperationsAsync[TService,TSimpleNameSyntax,TObjectCreationExpressionSyntax,TExpressionSyntax,TTypeDeclarationSyntax,TArgumentSyntax](<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.GenerateType.AbstractGenerateTypeService`6.Editor.GetOperationsAsync[TService,TSimpleNameSyntax,TObjectCreationExpressionSyntax,TExpressionSyntax,TTypeDeclarationSyntax,TArgumentSyntax](<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.GenerateType.AbstractGenerateTypeService`6.GenerateTypeCodeAction.ComputeOperationsAsync[TService,TSimpleNameSyntax,TObjectCreationExpressionSyntax,TExpressionSyntax,TTypeDeclarationSyntax,TArgumentSyntax](<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.GetPreviewOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedAction.GetPreviewResultAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedActionWithNestedFlavors.<>c__DisplayClass11_0.<GetPreviewAsync>b__0(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Extensions.IExtensionManagerExtensions.PerformFunctionAsync[T](<Unknown Parameters>)
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

I don't have a repro steps currently. :(
Hope the stacktrace is enough.

**Comment by jinujoseph** [comment]
@davidwengier dupe of https://github.com/dotnet/roslyn/issues/48973 ?

**Comment by davidwengier** [comment]
Possibly. @Youssef1313 what version of Visual Studio did you see this in? This stack is the same as #48973 which should have been fixed in 16.9 P2 by https://github.com/dotnet/roslyn/pull/48962

**Comment by Youssef1313** [comment]
Using 16.8.x.
closing
